src/client/index.js.wrapper.mustache

```diff
@@ -1,6 +1,6 @@
 (function () {
-    function initHammerheadClient(window, isIFrameWithoutSrc) {
-        // NOTE: the following script will be executed in the current window context. But when you call a script in an iFrame
+    function initHammerheadClient(window, isIframeWithoutSrc) {
+        // NOTE: the following script will be executed in the current window context. But when you call a script in an iframe
         // global vars are get from the top window. So, we should override the global vars that can have affect to our script.
```

Comment:
global vars are got from the top window. So, we should override global vars that can affect our script.